Source/WebCore/html/HTMLScriptElement.h

```diff
@@ -25,9 +25,12 @@
 
 #include "HTMLElement.h"
 #include "ScriptElement.h"
+#include "TrustedScript.h"
```

Comment:
I’d think we could compile `RefPtr<TrustedScript>` without including `TrustedScript.h`. We should be able to forward declare `TrustedScript`, and then include `TrustedScript.h` in the files that call these functions. An exception might be the bindings, which may need to compile calls to these functions without having an easy way to make it include `TrustedScript.h`. We want to work hard, though, to avoid adding more includes to headers because of the effect on build time.

Reply:
I believe the usage in the WEBCORE_EXPORT annotated method is what leads to this being required as an import rather than forward declaration. I can get the build error if that helps track down a potential solution?

Reply:
What’s the caller outside the bindings that requires `WEBCORE_EXPORT`? Maybe we can add an overload that is exported rather than exporting the version that takes a `std::variant`.

Reply:
...okay scratch that I've just tried again and it's working, so another change must have got it working (potentially it was the missing import in the WebKit legacy)